spec/views/collections/show.html.erb_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe 'collections/show.html.erb' do
+  let(:document) { SolrDocument.new({ id: 'xyz123z4', 'title_tesim' => ['Make Collections Great Again'] }) }
+  let(:ability) { double }
+  let(:presenter) { Sufia::CollectionPresenter.new(document, ability) }
+
+  before do
+    allow(document).to receive(:hydra_model).and_return(::Collection)
+    allow(controller).to receive(:current_user).and_return(stub_model(User))
+    assign(:presenter, presenter)
+    stub_template 'collection_title' => 'collection title'
+    stub_template 'collection_description' => 'collection description'
+    stub_template 'collections/show_descriptions' => 'descriptions'
+    stub_template 'collections/media_display' => 'media display'
+    stub_template 'search_form' => 'search form'
+    stub_template 'sort_and_per_page' => 'sort and per page'
+    stub_template 'document_list' => 'document list'
+    stub_template 'paginate' => 'paginate'
+    render
+  end
+
+  it 'displays an actions header' do
```

Comment:
Yeah, but right now this is all failing and I've got to abandon this branch possibly 'til after OR.